docs/guide/search/solr.md

```diff
@@ -22,12 +22,11 @@ The example presents a configuration with a single core. For configuring Solr in
 
 !!! note "Solr versions"
 
-    `ezplatform-solr-search-engine` v1 still supports Solr 6.
-    Since v2 `ezplatform-solr-search-engine` works only with Solr 7.
+    For v3 `ezplatform-solr-search-engine` works with Solr 7 and 8, using an supported LTS versions of Solr is recommended.
 
 ##### Solr 7
 
-Download and extract Solr. Solr Bundle 2 and higher supports Solr 7 *(currently tested and recommended with Solr 7.7)*:
+Download and extract Solr *(currently recommended with Solr 7.7LTS, so that is what example here is showing)*:
```

Comment:
```suggestion
Download and extract Solr *(a recommended major version is Solr 7.7.3)*:
```
7.7.3 is the last stable version of the 7.x series. 

To avoid confusion, I would use either this, or another tested and approved version, but give the exact version number, as assigned by Apache

Reply:
Major would refer to 7, so if we want to be specific patch release and say `7.7.3` I would maybe rather phrase it `*(7.7.3 + 8.6.3 tested and recommended at time of writing)*` But you end up having to update it quite often though, which is why I don't mention patch versions in doc usually but rather talk about which branches we support / recommend.

Reply:
"at the time of writing" is assumed, we do not have to state that.

Can we say  *(the most recent release in 7.7 or 8.6 series is recommended)*

Reply:
```suggestion
Download and extract Solr *(the most recent release in 7.7 or 8.6 series is recommended)*:
```